src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, applyMiddleware } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import thunk from 'redux-thunk';
+
+import logger from 'redux-logger';
+
+import reducer from './cryptodata/cryptodata';
+
+const store = createStore(
+  reducer,
+  composeWithDevTools(applyMiddleware(thunk, logger)),
+);
```

Comment:
- [ ] Please do not log redux data on the browser console as it may cause site security issues. Kindly clear the logged data displayed on the browser console.

Reply:
This change has been implemented well